pkg/spireentry/reconciler.go

```diff
@@ -307,7 +314,15 @@ func (r *entryReconciler) getUnsupportedFields(ctx context.Context) (map[spireap
 }
 
 func (r *entryReconciler) listClusterStaticEntries(ctx context.Context) ([]*ClusterStaticEntry, error) {
-	clusterStaticEntries, err := k8sapi.ListClusterStaticEntries(ctx, r.config.K8sClient)
+	var clusterStaticEntries []spirev1alpha1.ClusterStaticEntry
+	var err error
+	if r.config.K8sClient != nil {
+		clusterStaticEntries, err = k8sapi.ListClusterStaticEntries(ctx, r.config.K8sClient)
+	} else {
+		// FIXME prebuild / pass scheme?
+		scheme := runtime.NewScheme()
```

Comment:
are you planing to improve this schema? if it is not used until the very end may we create it there?

Reply:
Not intending on it. Moved.